forem/forem
# Fix readinglist count when article is unpublished

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

- Related Issue https://github.com/forem/forem/pull/15490
- Closes https://github.com/forem/forem/issues/18359

## Description

When an article is on a readinglist and the article becomes unpublished, the article is still including in the user's readinglist counter. In a [prior PR](https://github.com/forem/forem/pull/15490) we updated the reading list view to hide unpublished articles, this PR updates the counter. 

## QA Instructions, Screenshots, Recordings

* Create a post
* Click the bookmark icon to add the post to your readinglist
* See that your readinglist count (left sidebar) increments (this might require a page refresh)
* Unpublish the post
* Your readinglist should decrement (this might require a page refresh)

### UI accessibility concerns?

No accessibility chagnes

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![count](https://media.giphy.com/media/IXB6mQUgOqWQM/giphy.gif)
